scripts/rancher/user-actions.sh

```diff
@@ -55,33 +55,3 @@ rancher_update_password() {
     }' \
     "$rancherUrl/v3/users?action=changepassword"
 }
-
```

Comment:
Could we keep this method?

Rancher API is still needed sometimes and we may need it.

Reply:
I agree, I think we'll still need it... I'm now in a place where I probably need to do a hard refresh of the partner-charts repo before someone accesses it, to ensure that the newest versions are listed.

Reply:
I will put it back in there, and at the same time I want to try to use kubernetes CRDs approach as much as possible